admin/js/monk-admin.js

```diff
@@ -236,5 +236,16 @@
 				$( '#monk-menu-locations' ).remove();
 			}
 		}
+
```

Comment:
Since it will handle a multidimensional array, it's better use the `usort` function, with a lambda function